tensorflow/python/keras/engine/network.py

```diff
@@ -1059,24 +1056,21 @@ def save_weights(self, filepath, overwrite=True, save_format=None):
     `Layer` instances must be assigned to object attributes, typically in the
     constructor. See the documentation of `tf.train.Checkpoint` and
     `tf.keras.Model` for details.
-
     While the formats are the same, do not mix `save_weights` and
     `tf.train.Checkpoint`. Checkpoints saved by `Model.save_weights` should be
     loaded using `Model.load_weights`. Checkpoints saved using
     `tf.train.Checkpoint.save` should be restored using the corresponding
     `tf.train.Checkpoint.restore`. Prefer `tf.train.Checkpoint` over
     `save_weights` for training checkpoints.
-
     The TensorFlow format matches objects and variables by starting at a root
     object, `self` for `save_weights`, and greedily matching attribute
     names. For `Model.save` this is the `Model`, and for `Checkpoint.save` this
     is the `Checkpoint` even if the `Checkpoint` has a model attached. This
     means saving a `tf.keras.Model` using `save_weights` and loading into a
     `tf.train.Checkpoint` with a `Model` attached (or vice versa) will not match
     the `Model`'s variables. See the [guide to training
-    checkpoints](https://www.tensorflow.org/guide/checkpoint) for details
+    checkpoints](https://www.tensorflow.org/alpha/guide/checkpoints) for details
```

Comment:
Please don't change the URL to the alpha version, let's keep it to the version at head.

Is the URL at head broken?